relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -312,6 +315,10 @@ fn extract_shared_tags(event: &Event) -> BTreeMap<SpanTagKey, String> {
         }
         if let Some(country_code) = user.geo.value().and_then(|geo| geo.country_code.value()) {
             tags.insert(SpanTagKey::UserCountryCode, country_code.to_owned());
+            let country_to_subregion = get_country_to_subregion();
+            if let Some(subregion) = country_to_subregion.get(country_code.as_str()) {
+                tags.insert(SpanTagKey::UserSubregion, subregion.to_string().to_owned());
```

Comment:
Should we not use a machine readable identifier instead of a string here?

Reply:
Good call, perhaps we just use the subregion code instead?